pkg/transform/cluster_report_transform.go

```diff
@@ -148,6 +155,4 @@ func (e ClusterTransform) Extract() (Extraction, error) {
 }
 
 // Name returns a human readable name for the transform
-func (e ClusterTransform) Name() string {
-	return ClusterReportName
-}
+func (e ClusterTransform) Name() (ClusterReportName string) { return }
```

Comment:
This is not the equivalent.
The new Name() returns an empty value no matter what

Reply:
I'm going to add the tests, you are correct, sorry for the mess.